packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -9,37 +9,61 @@ import AccountWizard from './account-wizard.jsx';
 import AddOrManageAccounts from './add-or-manage-accounts.jsx';
 import FinishedSetCurrency from './finished-set-currency.jsx';
 import SignupErrorContent from './signup-error-content.jsx';
-import SuccessDialog from '../Modals/success-dialog.jsx';
+import StatusDialogContainer from './status-dialog-container.jsx';
 import 'Sass/account-wizard.scss';
 import 'Sass/real-account-signup.scss';
 
+const LoadingModal = () => <Loading is_fullscreen={false} />;
+
+const WizardHeading = ({ can_upgrade_to, currency, is_im_residence }) => {
+    if (!currency) {
+        return localize('Set a currency for your Real Account');
```

Comment:
why does this one return `localize()` and the others `<Localize />`?

Reply:
That certainly was a whoopsie. :stuck_out_tongue_winking_eye: